src/Data/Connection/PostObjectConnectionResolver.php

```diff
@@ -41,7 +41,14 @@ public function __construct( $source, $args, $context, $info, $post_type ) {
 		/**
 		 * Set the post type for the resolver
 		 */
-		$this->post_type = $post_type;
+		if ( 'revision' === $post_type || 'attachment' === $post_type ) {
+			$this->post_type = $post_type;
```

Comment:
I'm confused... wouldn't you _not_ want to set this as the `post_type` for these two post types?

Reply:
@CodeProKid no, you can still query these, but not as an array. 

For example, the `revisions` connections use this resolver and set the `post_type => 'revision'` so connections to singular Post Types, such as `revision` or `attachment` should still work. 

However, queries to plural post types should not include `revisions` or `attachments` as they're rules for resolving are different (they're always 'inherit' status).

Reply:
Ahhhh, OK. I get the logic here now. Such nuance. 🐶

Reply:
ya, I added a comment. It is kinda weird, but such is life when it comes to how WordPress handles a lot of things 😆